src/compile/layoutsize/parse.ts

```diff
@@ -97,7 +97,7 @@ export function parseUnitLayoutSize(model: UnitModel) {
   for (const channel of POSITION_SCALE_CHANNELS) {
     const sizeType = getSizeChannel(channel);
 
-    if (size[sizeType]) {
+    if (size[sizeType] !== undefined) {
```

Comment:
Should this be using `size[sizeType] != null` or should we not catch null and only catch undefined? How is this different from the logic above then?